ibis/expr/datatypes.py

```diff
@@ -801,7 +801,7 @@ class MultiPolygon(GeoSpatial):
     __slots__ = ()
 
 
-class UUID(String):
+class UUID(DataType):
```

Comment:
This should probably be in a separate PR since it's technically a breaking change. All of the `datatypes.py` classes are public at the moment.

Reply:
Removed the UUID change and will push up a fast-follow PR for this change